### ISSUE TITLE
Image explaining the use of alt texts has no alt text

### ISSUE THREAD

**Issue by kvnloughead** [mdn/yari#5228]
This image [here](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML) of the Mozilla red dinosaur illustrating how alt texts appear over images on hover has no alt text. 

![image](https://user-images.githubusercontent.com/37159177/152055022-bb9e638b-4cab-4074-ac3b-5a5c0d46dceb.png)

Suggested alt text:

> Alt text tooltip reading "The Mozilla red dinosaur" hovering over the Mozilla red dinosaur.

**Comment by SphinxKnight** [comment]
@mdn/core I think this is to be moved over to mdn/content
